doc/source/serve/production-guide/config.md

```diff
@@ -199,3 +199,5 @@ Serve will set `num_replicas=5`, using the config file value, and `max_concurren
 :::{tip}
 Remember that `ray_actor_options` counts as a single setting. The entire `ray_actor_options` dictionary in the config file overrides the entire `ray_actor_options` dictionary from the graph code. If there are individual options within `ray_actor_options` (e.g. `runtime_env`, `num_gpus`, `memory`) that are set in the code but not in the config, Serve still won't use the code settings if the config has a `ray_actor_options` dictionary. It will treat these missing options as though the user never set them and will use defaults instead. This dictionary overriding behavior also applies to `user_config`.
 :::
+
+[RayService]: https://ray-project.github.io/kuberay/guidance/rayservice/
```

Comment:
can you link to the k8s production guide section instead? want to keep people on ray docs where possible

Reply:
I think I linked out to this in other places more than I should have in retrospect...